model/src/main/java/oracle/kubernetes/weblogic/domain/v1/DomainSpec.java

```diff
@@ -85,13 +86,25 @@
   private Integer asPort;
 
   /**
+   * The in-pod name of the directory to store the domain, node manager, server logs, and server
+   * .out files in.
+   */
+  @SerializedName("logHome")
+  @Expose
+  private String logHome;
+  /** Whether to include the server .out file to the pod's stdout. Default is true. */
+  @SerializedName("includeServerOutInPodLog")
+  @Expose
+  private String includeServerOutInPodLog;
+
+  /*
    * The WebLogic Docker image.
    *
-   * <p>Defaults to store/oracle/weblogic:12.2.1.3.
+   * <p>Defaults to store/oracle/weblogic:19.1.0.0
```

Comment:
And, again :)

Reply:
https://github.com/oracle/weblogic-kubernetes-operator/pull/489/commits/4de0cf306c42f6d40fdbd24f1ea082930b83ec20